api/segments/models.py

```diff
@@ -60,24 +59,30 @@ class Segment(
     # This defaults to 1 for newly created segments.
     version = models.IntegerField(null=True)
 
-    # The related_name is not useful without specifying all_objects as a manager.
     version_of = models.ForeignKey(
         "self",
         on_delete=models.CASCADE,
         related_name="versioned_segments",
         null=True,
         blank=True,
     )
+
+    change_request = models.ForeignKey(
+        "workflows_core.ChangeRequest",
+        on_delete=models.CASCADE,
+        null=True,
+        related_name="segments",
+    )
```

Comment:
let's add `blank=True` here too in case it appears in the django admin anywhere

Reply:
Good idea. Implemented.